doc/xds-test-descriptions.md

```diff
@@ -0,0 +1,307 @@
+# xDS (Load-Balancing) Interop Test Case Descriptions
+
+Client and server use [test.proto](../src/proto/grpc/testing/test.proto).
+
+## Server
+
+The code for the xDS test server can be found
+[here](https://github.com/grpc/grpc-java/blob/master/interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java)
+(Java; other language implementations are in progress). are in progress).
+
+Server should accept these arguments:
+
+*   --port=PORT
+    *   The port the server will run on.
+
```

Comment:
What about "server_id"?

Reply:
This is actually not required, and convenient to not use, for OSS tests. The hostname on GCP corresponds nicely to the instance name, and so the instance template used for the server backends can just start a gRPC server, without needing an `id` parameter from the test script (otherwise `run_xds_tests.py` would have to start up the instances and separately pass a parameter to each server, likely at start up time - instance templates cannot have params). This wouldn't be too inconvenient given the final structure of `run_xds_tests.py`, but it's not necessary right now.

Internally, we currently use `server_id` (and may have to continue to do so) to distinguish the backends for these tests, but I don't think it needs to be specced out here at the moment.